plugin/pkg/scheduler/scheduler.go

```diff
@@ -121,16 +121,6 @@ type Config struct {
 	StopEverything chan struct{}
 }
 
-// New returns a new scheduler.
-// TODO replace this with NewFromConfigurator.
-func New(c *Config) *Scheduler {
-	s := &Scheduler{
-		config: c,
-	}
-	metrics.Register()
```

Comment:
`New` is replaced by `NewFromConfigurator` now. `New` is not use in kube-scheduler but tests, I'm updating code for them.